src/index.js

```diff
@@ -40,11 +40,10 @@ async function run() {
   }
 
   // Find the two most recent tags
-  const { data: tags } = await octokit.rest.repos.listTags({
-    owner,
-    repo,
-    per_page: 10,
-  });
+  // Using https://octokit.github.io/rest.js/v19#pagination
+  const { data: tags } = await octokit.paginate("GET /repos/{owner}/{repo}/tags",
```

Comment:
This actually needed to be changed to just `const data = ...` but I fixed it.

v1.1.1 is released, thanks for the PR!

Reply:
Oh yes of course I wrote it good locally and then I failed the copy/paste .. 
Thank you !